tests/device_tests/ethereum/test_sign_verify_message.py

```diff
@@ -0,0 +1,92 @@
+# This file is part of the Trezor project.
+#
+# Copyright (C) 2012-2019 SatoshiLabs and contributors
+#
+# This library is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License version 3
+# as published by the Free Software Foundation.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the License along with this library.
+# If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.
+
+import pytest
+
+from trezorlib import ethereum
+from trezorlib.tools import parse_path
+
+from ...common import parametrize_using_common_fixtures
+
+pytestmark = [pytest.mark.altcoin, pytest.mark.ethereum]
+
+
+@parametrize_using_common_fixtures(
+    "ethereum/signmessage.json",
```

Comment:
for future reference (let's not touch this PR now):

This looks like it would fit on a single line.
The reason it's formatted like this is the trailing `,` -- if you have a trailing comma in the argument list, `black` will put each argument on a separate line and keep it that way. If you removed the comma, `black` would create a single line from this.

Generally we want to keep things on one line if possible.